web-build/Dockerfile.spx

```
@@ -0,0 +1,30 @@
+#ddev-generated
+RUN <<EOS
```

Hadolint (2.12.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)